src/routes/(console)/organization-[organization]/billing/paymentHistory.svelte

```diff
@@ -134,12 +134,12 @@
                                     <ActionMenu.Item.Anchor
                                         leadingIcon={IconExternalLink}
                                         external
-                                        href={`${base}/organizations/${page.params.organization}/invoices/${invoice.$id}/view`}>
```

Comment:
svelte diagnostic error because of base. I also have a fix over https://github.com/appwrite/console/pull/1917